src/Microsoft.ML.Transforms/Text/StopWordsRemovingTransformer.cs

```diff
@@ -438,6 +441,45 @@ protected override Delegate MakeGetter(DataViewRow input, int iinfo, Func<int, b
                 return del;
             }
 
+            public void SaveAsOnnx(OnnxContext ctx)
+            {
+                const int minimumOpSetVersion = 9;
+                ctx.CheckOpSetVersion(minimumOpSetVersion, LoaderSignature);
+
+                for (int i = 0; i < _parent.ColumnPairs.Length; i++)
+                {
+                    var srcVariableName = ctx.GetVariableName(_parent.ColumnPairs[i].inputColumnName);
+                    if (!ctx.ContainsColumn(srcVariableName))
+                        continue;
+                    var dstVariableName = ctx.AddIntermediateVariable(_types[i], _parent.ColumnPairs[i].outputColumnName);
+                    SaveAsOnnxCore(ctx, i, srcVariableName, dstVariableName);
+                }
+            }
+
+            private void SaveAsOnnxCore(OnnxContext ctx, int iinfo, string srcVariableName, string dstVariableName)
+            {
+                var opType = "Squeeze";
+                var squeezeOutput = ctx.AddIntermediateVariable(null, "SqueezeOutput", true);
+                var node = ctx.CreateNode(opType, srcVariableName, squeezeOutput, ctx.GetNodeName(opType), "");
```

Comment:
Is it possible to avoid skipping the shape and type addition?

Reply:
The reason I skipped the shape is because the shape of the tokenized word vector is not known prior to inference time. The number of words tokenized may be any number.

Reply:
Other text transformers that also infer shape: 
- https://github.com/dotnet/machinelearning/blob/master/src/Microsoft.ML.Transforms/Text/WordTokenizing.cs
- https://github.com/dotnet/machinelearning/blob/master/src/Microsoft.ML.Transforms/Text/TokenizingByCharacters.cs